include/Core/DB/TableManager.php

```diff
@@ -239,4 +239,19 @@ public function fetch_log_item_by_id( $id ) {
 		$query      = $wpdb->prepare( 'SELECT * FROM ' . $table_name . ' WHERE id = %d', $id );
 		return $wpdb->get_results( $query );
 	}
+
+    /**
+     * Deletes Email Logs older than the specified interval.
+     *
+     * @param $interval_in_days
```

Comment:
It might be better to include data type and a very short description of the parameters.